tsconfig-base.json

```
@@ -0,0 +1,21 @@
+{
+    "$schema": "https://json.schemastore.org/tsconfig",
+    "display": "Node 14",
+  
+    "compilerOptions": {
+      "lib": ["es2020"],
+      "module": "commonjs",
```

```suggestion
      "module": "ES2020",
```
I think this needs to be es2020 now that you've changed the module type in package.json